docker/makeSupportedBrowsersScript.js

```diff
@@ -19,8 +19,8 @@ const isModern = ${r}.test(userAgent);
 process.stdout.write(isModern ? 'modern' : 'legacy');
 `;
 
-if (outputFileName) fs.writeFile(outputFileName, program, 'utf8', err => {
-  if (err) throw err;
-});
-
-else process.stdout.write(program);
\ No newline at end of file
+if (outputFileName)
+  fs.writeFile(outputFileName, program, 'utf8', (err) => {
+    if (err) throw err;
+  });
+else process.stdout.write(program);
```

Comment:
Huh, I'm curious how these formatting changes came about? Obviously trivial, just curious!

Reply:
no idea! lol i assumed i must have opened it by accident at some point. I thought about reverting but since it looked like only formatting changes i didn't worry about it haha